web-client/src/views/CreateOrder/TextEditor.tsx

```diff
@@ -19,13 +21,40 @@ const ReactQuill = React.lazy(async () => {
   return { default: reactQuill };
 });
 
+// Quill does two things to render indentations:
+// 1) Adds padding-left so that the richText (and therefore the pdf) has indentations.
+// 2) Adds custom ql-indent-{level} classes to display the indentation in the Quill editor itself.
+// As part of the richText, 1 is automatically preserved across save and reload.
+// As part of the client-side Quill text rendering, 2 is *not* preserved across save and reload,
+// so even though the indentations are "there" in the richText, the user won't see them in the editor.
+// We call addQuillIndentationClasses on loading the text into the editor in order to correct that.
```

Comment:
I'm not a fan of long-winded comments, but I spent enough time mucking through this that I think it would be more helpful than hurtful to other developers.

Reply:
I appreciate the detail!